nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-canvas-utils.js

```diff
@@ -412,9 +412,9 @@
 
                 // create object whose keys are the parameter name and the values are the parameter values
                 var paramsObject = {};
-                params.forEach(function (v, k) {
-                    paramsObject[k] = v;
-                });
+                for(var pair of params.entries()) {
```

Comment:
Yes, this works in build and in bringing up the console!  Thanks for the update.